src/components/Header/Header.component.jsx

```diff
@@ -8,18 +8,18 @@ import LoginAvatar from '../LoginAvatar';
 const Header = () => (
   <Styled.Header>
     <ul>
-      <Styled.LeftItem>
+      <Styled.FloatedItem float="left">
```

Comment:
At the import you can do like:
import { Header, FloatedItem } from './styled';

then at the JSX use it like:
<Header>
  <ul> 
    <FloatedItem ...>